polkadot/scripts/packaging/polkadot.service

```diff
@@ -5,6 +5,7 @@ Documentation=https://github.com/paritytech/polkadot
 
 [Service]
 EnvironmentFile=-/etc/default/polkadot
+ExecStartPre=/usr/bin/polkadot key generate-node-key --default-base-path
```

Comment:
It is executed on every restart. If the key is already there, wouldn't it re-generate the key on every restart then?

Reply:
Okay, I'm not sure I follow then. You execute the key generation procedure on every node restart, and it generates a new key if it's not there already. How is it different from the current behavior we're trying to avoid?

Reply:
Yeah, I think you are right here I'm creating the same problem I'm trying to solve, good catch!